droidlet/dashboard/web/src/StateManager.js

```diff
@@ -131,6 +131,7 @@ class StateManager {
       objects: false,
       pose: false,
     };
+    this.useDesktopComponentOnMobile = false; // switch to use either desktop or mobile annotation on mobile device
```

Comment:
Can you add a TODO or NOTE that the mobile annotation UX on mobile is incomplete?